elixir/stalin_sort.ex

```diff
@@ -0,0 +1,13 @@
+defmodule StalinSort do
+  def sort(list) do
+    list
+      |> Enum.with_index
+      |> Enum.reduce([], fn({number, index}, acc) -> 
+        is_ordered = index == 0 || number > Enum.at(acc, -1)
+        case is_ordered do
+            true ->  acc ++ [number]
+            false -> acc
+        end
+      end)
+  end
+end
```

Comment:
I don't know much about Elixir, but there is a exporting function? If I want to use this code in my project, how do I proceed? Is needed to export a function from a file to import it in another one?

Reply:
A module in elixir works similar to namespaces, all the functions are defined inside of modules and the compiler scans all of the directory contents of any .ex files so you don't need to explicitly export then.

To call the `sort` function you just need to use the syntax `StalinSort.sort(list)` or
you can also import this function inside of other files without it's module's name by using `import`:
``` elixir
defmodule Test do
  import StalinSort

  def sort_stuff do
    sort([1, 2, 3, 4, 5])
  end
end
```